samples/Compute/deploy-dsc-extension/azurepolicy.rules.json

```diff
@@ -0,0 +1,286 @@
+
```

Comment:
Missing opening {, that is why the test is failing

Reply:
fixed